integrations/cross_platform/yocto.rst

```diff
@@ -0,0 +1,271 @@
+.. _yocto_integration:
+
+
+|yocto_logo| Yocto
+__________________
+
+The `Yocto Project`_ is an open-source project that delivers a set of tools that create operating system images for embedded Linux systems.
+The Yocto Project tools are based on the `OpenEmbedded`_ project, which uses the BitBake build tool, to construct complete Linux images.
+
+Brief introduction to Yocto
+===========================
+
+Yocto supports several Linux host distributions and it also provides a way to install the correct version of these tools by either
+downloading a buildtools-tarball or building one on a supported machine. This allows virtually any Linux distribution to be able to run
+Yocto, and also makes sure that it will be possible to replicate your Yocto build system in the future. The Yocto Project build system also
+isolates itself from the host distribution's C library, which makes it possible to share build caches between different distributions and
+also helps in future-proofing the build system.
+
+Creating Conan packages with Yocto's SDK
+========================================
+
+Setting up a Yocto SDK
```

Comment:
New section How it works: Describe (and introduce a schema image) of the whole picture with Conan and Yocto. Where is the developer, where is artifactory, where is the poky project, what is traveling to/from Artifactory, why we want these: 
- Developer experience is the same, build for desktop, then swtich to yocto sdk and build for target
- The poky image building is fast by pulling directly the app from Artifactory. It can be automated in a CI, explain.

Reply:
yes, this was missing. I have added this at the top to have in mind the general picture. Thanks!